packages/AutoTDD-GUI.package/ATDDGitHubActionResults.class/instance/simplifySmalltalkCiLogCheckBoxLabel.st

```diff
@@ -0,0 +1,4 @@
+building
+simplifySmalltalkCiLogCheckBoxLabel
+	
+	^ 'Simplify SmalltalkCI Log'
```

Comment:
```suggestion
accessing
simplifyLogCheckBoxLabel
	
	^ 'Simplify ' , _fetch String here instead_ , ' Log'
```
I felt the names were a little too long to support readability. Although the parsing is in the same class i would not include any info on implementation (SmalltalkCiLog vs Log) in GUI code

Would put this String in this class to allow for reusability for potential simplification option of travis status log (then we could move all this code to ATDDRemoteResults)